lesson12/index.js

```diff
@@ -0,0 +1,68 @@
+const ul = document.getElementById('js-ul');
+const div = document.getElementById('js-wrapper');
+const button = document.getElementById("js-button");
+
+
+function addLoading() {
+  const loading = document.createElement('img');
+  loading.src = "./loading-circle.gif";
+  loading.id = "loading";
+  ul.appendChild(loading);
+}
+
+function removeLoading() {
+  const loading = document.getElementById('loading');
+  loading.remove();
+}
+
+
+async function fetchData() {
+  addLoading();
```

Comment:
Thanks for the review!

I changed the specification to execute addLoading, removeLoading, and fetchData in the tryCreate function.
The reason is that I thought it would be better than adding a new function.
What do you think?

Reply:
IMO

I think `createLists` is not part of `fetchData` as well because that name means just like a getting data.

It might be good to call `createLists` in the try part of `tryCreate`.
Actually we shouldn't execute `createList` in try part of `tryCreate` because if error happens, we can't tell whether the error is from "fetchData" or "createLists"(I was taught this before from other student). But in this case, you implemented Error on each function, so if you implement error message more clearly I think this is one of the way.
Or I think just to change the name of `fetchData` might be better if you don't use `tryCreate` and make other function :)

Reply:
Thanks for the review and the advice!

Added init() function.

```
if(response !== undefined) {
    createLists(response);
  }
```
Use the if statement to execute createLists() only if no error occurs in fetchData.


f4a03b5

Reply:
It looks good to me! :)